### PR TITLE
Updated README 

### PR DESCRIPTION
**Pull request by hansenms**
## Description
Updated README to mention Azure API for FHIR and reference the docs.microsoft.com pages

## Related issues
N/A
## Testing
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,33 +10,37 @@ With data in the FHIR format, the FHIR Server for Azure enables developers to qu
...
+This open source project is fully backed by the Microsoft Healthcare team, but we know that this project will only get better with your feedback and contributions. We are leading the development of this code base, and test builds and deployments daily.
+
+There is also a managed offering in Azure called the [Azure API for FHIR](https://azure.microsoft.com/services/azure-api-for-fhir/). This Platform as a Service (PaaS) FHIR server is backed by the open source project in this repository and it offers a turn key solution to provisioning a compliant, secure FHIR service.
 
 # Documentation
 
-- [Deployment](docs/DefaultDeployment.md): Describes how to deploy FHIR Server for Azure. 
+- [Open Source Deployment](https://docs.microsoft.com/en-us/azure/healthcare-apis/fhir-oss-portal-quickstart): Describes how to deploy FHIR Server for Azure.
+- [PaaS Service Deployment](https://docs.microsoft.com/en-us/azure/healthcare-apis/fhir-paas-portal-quickstart): Describes how to deploy the Azure API for FHIR.
```
Comment (cunninghamjc) [inline README.md]: Nit: the line above says "deploy FHIR Server for Azure" whereas here we say "deploy the Azure API for FHIR."  Is the extra "the" here intentional?

File: README.md [hunk 2]
```diff
@@ -10,33 +10,37 @@ With data in the FHIR format, the FHIR Server for Azure enables developers to qu
...
 - [Roles](docs/Roles.md): Describes how the FHIR Server for Azure role-based access control (RBAC) system works.
 - [Search](docs/Search.md): Describes how search is implemented for the FHIR Server for Azure.
 - [Debugging](docs/HowToDebug.md): Describes how to debug FHIR Server for Azure using Visual Studio.
 - [Testing with Postman](docs/PostmanTesting.md): Describes how to use Postman for testing FHIR Server for Azure.
 
-## More Information 
+## More Information
 
-* [FHIR Server for Azure Demo by Michael Hansen, Senior Program Manager](https://github.com/hansenms/FhirDemo)
-* [Features](docs/Features.md): This document lists the main features of the FHIR Server for Azure.
```
Comment (cunninghamjc) [inline README.md]: Should we remove Features.md?

File: README.md [hunk 3]
```diff
@@ -10,33 +10,37 @@ With data in the FHIR format, the FHIR Server for Azure enables developers to qu
...
 
-- [Deployment](docs/DefaultDeployment.md): Describes how to deploy FHIR Server for Azure. 
+- [Open Source Deployment](https://docs.microsoft.com/en-us/azure/healthcare-apis/fhir-oss-portal-quickstart): Describes how to deploy FHIR Server for Azure.
+- [PaaS Service Deployment](https://docs.microsoft.com/en-us/azure/healthcare-apis/fhir-paas-portal-quickstart): Describes how to deploy the Azure API for FHIR.
 - [Azure Active Directory Application Registrations](docs/PortalAppRegistration.md): Describes how to configure Azure Active Directory (AAD) for use with FHIR Server for Azure.
 - [Authentication](docs/Authentication.md): Describes the authentication settings for the FHIR server and how to make use of it in development and test scenarios.
 - [Roles](docs/Roles.md): Describes how the FHIR Server for Azure role-based access control (RBAC) system works.
 - [Search](docs/Search.md): Describes how search is implemented for the FHIR Server for Azure.
 - [Debugging](docs/HowToDebug.md): Describes how to debug FHIR Server for Azure using Visual Studio.
 - [Testing with Postman](docs/PostmanTesting.md): Describes how to use Postman for testing FHIR Server for Azure.
```
Comment (cunninghamjc) [inline README.md]: Should we point to https://docs.microsoft.com/en-us/azure/healthcare-apis/access-fhir-postman-tutorial?

File: README.md (hunk 2 above)
Comment (hansenms) [reply README.md]: Removed it.

File: README.md (hunk 3 above)
Comment (hansenms) [reply README.md]: Added tutorial.

**Pull request merged**